<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisUI</class>
 <widget class="QWidget" name="AnalysisUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>961</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold Condensed</family>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  Start Analysis</string>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>941</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="YourTeammates">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold Condensed</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>  Your Teammates</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="MemberTitle">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold Condensed</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>   Member 1:                                Member 2:                              Member 3:                               Member 4:                              Member 5:                  </string>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="Member1">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Member2">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Member3">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Member4">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="Member5">
        <property name="font">
         <font>
          <family>Bahnschrift SemiBold Condensed</family>
          <pointsize>15</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>671</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="AddSet">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold Condensed</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add this Team</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Confirm">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold Condensed</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cancel">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold Condensed</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/image/logo1.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>941</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
